packages/fxa-auth-server/lib/senders/emails/partials/automatedEmailResetPassword/index.mjml

```diff
@@ -1,15 +1,21 @@
-<%# This Source Code Form is subject to the terms of the Mozilla Public
-  # License, v. 2.0. If a copy of the MPL was not distributed with this
-  # file, You can obtain one at http://mozilla.org/MPL/2.0/. %>
+<%# This Source Code Form is subject to the terms of the Mozilla Public #
+License, v. 2.0. If a copy of the MPL was not distributed with this # file, You
+can obtain one at http://mozilla.org/MPL/2.0/. %>
```

Comment:
👀 I'm guessing prettier did this? I think we should consider disabling prettier for MJML files since it's producing less-than-optimal results.

Reply:
Yeah, this was prettier's doing. I think if possible we might consider changing these MJML files to EJS files